lib/chef/mixin/windows_architecture_helper.rb

```diff
@@ -43,6 +43,14 @@ def wow64_architecture_override_required?(node, desired_architecture)
       end
 
       def with_os_architecture(node)
+        node ||= begin
```

Comment:
when is this needed? ohai provides what appears to be an appropriate node[:kernel][:machine] on my workstation